### PR TITLE
Allow expense tracker to take in duplicate expenses. Debug for massedit command.

### PR DESCRIPTION
**Pull request by jcjxwy**

### CODE REVIEW

File: docs/team/jiangchen.adoc [hunk 1]
```diff
@@ -32,8 +32,8 @@ as it required changes to existing commands.
 * *Other contributions*:
 
 ** Enhancements to existing features:
-** Refactor original AddressBook and Person class into ExpenseTracker and Expense class.
-** Update sample entries.
+** Refactors original AddressBook and Person class into ExpenseTracker and Expense class.
```
Comment (Snookerballs) [inline docs/team/jiangchen.adoc]: Maybe use past tense i.e Refactored/Updated? Current tense being used might be grammatically incorrect

File: src/test/java/seedu/address/model/expense/ExpenseListTest.java [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+    @Test
+    public void remove_existingExpense_removesExpense() {
+        expenseList.add(SCHOOLFEE);
+        expenseList.remove(SCHOOLFEE);
+        ExpenseList expectedExpenseList = new ExpenseList();
+        assertEquals(expectedExpenseList, expenseList);
+    }
+
+    @Test
+    public void setExpenses_nullUniqueExpenseList_throwsNullPointerException() {
```
Comment (Snookerballs) [inline src/test/java/seedu/address/model/expense/ExpenseListTest.java]: remove 'unique' from method name

File: src/test/java/seedu/address/model/expense/ExpenseListTest.java [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+        thrown.expect(NullPointerException.class);
+        expenseList.setExpenses((ExpenseList) null);
+    }
+
+    @Test
+    public void setExpenses_uniqueExpenseList_replacesOwnListWithProvidedUniqueExpenseList() {
```
Comment (Snookerballs) [inline src/test/java/seedu/address/model/expense/ExpenseListTest.java]: remove 'unique' from method name

File: docs/UserGuide.adoc [hunk 4]
```diff
@@ -239,29 +239,28 @@ Format: `edit INDEX [n/EXPENSE_NAME] [$/COST] [c/CATEGORY] [d/DATE] [t/TAG...]`
...
 Existing values of the expense will be edited according to the value of the parameters +
 
-Format: `massedit [n/EXPENSE_NAME] [$/COST] [c/CATEGORY] [d/DATE] [t/TAG...] -> [n/EXPENSE_NAME] [$/COST] [c/CATEGORY] [d/DATE] [t/TAG...]`
+Format: `massedit [n/EXPENSE_NAME] [$/COST] [c/CATEGORY] [d/DATE] [t/TAG...] \-> [n/EXPENSE_NAME] [$/COST] [c/CATEGORY] [d/DATE] [t/TAG...]`
 
 ****
 *Examples:*
 
-* `massedit c/school -> c/food` +
+* `massedit c/school \-> c/food` +
```
Comment (winsonhys) [inline docs/UserGuide.adoc]: Is the slash supposed to be there

**Review by winsonhys — CHANGES_REQUESTED** [review]

File: docs/UserGuide.adoc (hunk 4 above)
Comment (jcjxwy) [reply docs/UserGuide.adoc]: Yes, this is to ensure that the "->" will not become "→" automatically in UserGuide.

**Review by winsonhys — APPROVED** [review]

**Pull request merged**